packages/api/src/controllers/user.ts

```diff
@@ -60,6 +62,39 @@ async function findUserByEmail(email: string, useReplica = true) {
   return users[0];
 }
 
+const frontendUrl = (
+  {
+    headers: { "x-forwarded-proto": proto },
+    config: { frontendDomain },
+  }: Request,
+  path: string
+) => `${proto || "http"}://${frontendDomain}${path}`;
+
+const unsubscribeUrl = (req: Request) => frontendUrl(req, "/contact");
+
+export async function suspendUserStreams(
+  req: Request,
+  userId: string,
+  suspended: boolean
+): Promise<void> {
+  const [streams] = await db.stream.find({ userId });
+  for (const stream of streams) {
```

Comment:
Nice! Once for all of them

Reply:
Hopefully. The only problem I guess is that it's not really reliable, which is why we need that MistUtilNukeProc or something like that, running directly on the container :/

But maybe the problem is just something on our side, idk.